### PR TITLE
Clarify the source and usage of postData variable.

### PR DESCRIPTION
**Pull request by niamleeson**
I think newcomers will benefit from an extended explanation of the block component.

### CODE REVIEW

File: source/localizable/components/wrapping-content-in-a-component.md [hunk 1]
```diff
@@ -93,6 +93,14 @@ Once yielded the data can be accessed within wrapped content by referencing the
 Finally we want to share the model of the data a user fills out for the post within our `blog-post` and body components.
 To share the `postData` object with the new body component, you can add arguments to the component helper.
```
Comment (toddjordan) [inline source/localizable/components/wrapping-content-in-a-component.md]: "Finally, we want to share the the blog text that a user fills out for the post within our `blog-post` and body components. To share the blog text with the new body component, we'll add a `postData` argument to the component helper."

File: source/localizable/components/wrapping-content-in-a-component.md [hunk 2]
```diff
@@ -93,6 +93,14 @@ Once yielded the data can be accessed within wrapped content by referencing the
...
 
+```app/components/blog-post.js
```
Comment (toddjordan) [inline source/localizable/components/wrapping-content-in-a-component.md]: I think binding postData directly into the blog-post component declaration above would be better than giving it internal state.

File: source/localizable/components/wrapping-content-in-a-component.md [hunk 3]
```diff
@@ -105,9 +113,26 @@ When `{{post.body}}` is instantiated, it will have both the edit style and the `
 ```app/templates/index.hbs
 {{#blog-post editStyle="markdown-style" as |post|}}
   <p class="author">by {{author}}</p>
-  {{post.body editStyle="compact-style"}}
+  {{post.body bodyStyle="compact-style"}}
+{{/blog-post}}
+```
+
+Alternatively, `postData` can be passed to `blog-post` at the block level if it was made available outside the component block as `myText`.
```
Comment (toddjordan) [inline source/localizable/components/wrapping-content-in-a-component.md]: I don't think we need to give both alternatives here.  Just use this as the main example (like commented above).  It distracts from what the main point of the section is.

File: source/localizable/components/wrapping-content-in-a-component.md (hunk 1 above)
Comment (niamleeson) [inline source/localizable/components/wrapping-content-in-a-component.md]: Thank you for the improvement. The text has been updated.

File: source/localizable/components/wrapping-content-in-a-component.md (hunk 3 above)
Comment (niamleeson) [inline source/localizable/components/wrapping-content-in-a-component.md]: Removed the alternative as you suggested.

**Comment by toddjordan** [comment]
Thanks, @niamleeson 

@homu r+

**Comment by toddjordan** [comment]
#1738

**Pull request merged**